website/js/title-analysis.js

```diff
@@ -1,19 +1,19 @@
-function p2AddNewsCard(media, header, detail) {
-    var selector = "#2th-collection .card-container ." + media;
-    var content = "<div class=\"card\">\
+function titleAnalysisAddNewsCard(media, header, detail) {
+  var selector = "#2th-collection .card-container ." + media;
+  var content = "<div class=\"card\">\
```

Comment:
It is recommended to use single quotes `''` for strings in JS. And the whitespace at the beginning of each line can't be safely stripped at compile time; whitespace after the slash will result in tricky errors. Use + for string concatenation instead: